### PR TITLE
Remove functional transformers from the ppl to fix sklearn warnings

### PR DESCRIPTION
**Pull request by lucywang000**
This should address #15 . I have tested it with sklearn 0.19.1 and 0.20.0

Related discussion: https://github.com/TeamHG-Memex/soft404/issues/15#issuecomment-424699517

Also fix the tests after html_text 0.4 breaks the tests.

### CODE REVIEW

**Comment by lucywang000** [comment]
ping @lopuhin @kmike

**Comment by lucywang000** [comment]
Travis fails with strange errors in py35-train and py36-train tox env:

```
        if save:
            pipeline = Pipeline([('vec', vec)] + text_pipeline.steps)
>           Soft404Classifier.save_model(save, pipeline)
E           NameError: name 'Soft404Classifier' is not defined
```

Any ideas?

**Comment by lopuhin** [comment]
@lucywang000 thanks for the PR! Regarding the error, I think it happens because import of ``Soft404Classifier`` was removed here https://github.com/TeamHG-Memex/soft404/pull/16/files#diff-a5c400537b9dda586f39e3b735026061L28 but it's still used in the code.

**Comment by lucywang000** [comment]
@lopuhin I missed that - thanks for catching!

I think it's fixed now.

File: soft404/utils.py [hunk 1]
```diff
@@ -40,7 +40,7 @@ def get_text_blocks(tree):
 
 def html_to_item(html):
     sel = cleaned_selector(html)
-    text = selector_to_text(sel)
+    text = selector_to_text(sel, guess_layout=False)
```
[the remark this answers is not in the document]
Comment (lucywang000) [reply soft404/utils.py]: Agree, updated.

**Comment by lopuhin** [comment]
Sorry, I wanted to do a regular merge, didn't notice it was set to squash.

**Comment by lucywang000** [comment]
@lopuhin Thanks for the discussion & review!

**Pull request merged**